src/components/Menu/Menu.stories.tsx

```diff
@@ -2,34 +2,71 @@
 
 import React from 'react';
 import { Meta, StoryFn } from '@storybook/react';
+import { MenuItem } from '@components/MenuItem';
 import { Menu, MenuProps } from './Menu';
-import { BasicMenuComponent, ITEMS, ITEMS_WITH_LINKS, Item, NoTriggerMenuComponent } from './utils/componentMocks';
+import {
+    BASIC_MENU_WITH_LINKS_CODE_SNIPPET,
+    BASIC_MENU_WITH_ONCLICK_CODE_SNIPPET,
+    BasicMenuComponent,
+    COMPONENT_STRUCTURE_CODE_SNIPPET,
+    ITEMS,
+    ITEMS_WITH_LINKS,
+    Item,
+} from './utils/componentMocks';
 
 export default {
     title: 'Components/Menu',
     component: Menu,
     tags: ['autodocs'],
 } as Meta<MenuProps>;
 
-const TemplateWithoutTrigger: StoryFn<MenuProps & { items: Item[] }> = (args) => (
+const TemplateWithoutTrigger: StoryFn = () => (
     <div className="tw-p-4">
-        <NoTriggerMenuComponent {...args} />
+        <Menu>
+            <MenuItem link="https://www.frontify.com">Item 1</MenuItem>
```

Comment:
I would remove the link from the actual story implementation. I was testing the component and clicked on the ltems. It can be annoying to get redirected to Frontify. Especially because it does not open on another tab.